components/dialog/action-buttons.md

```diff
@@ -73,3 +73,7 @@ The following example demonstrates all supported layout options for the Dialog a
     }
 }
 ````
+
+## See Also
+
+* [KB: Keep the content in the DOM when the Window is closed]({%slug window-kb-keep-content-when-closed%})
```

Comment:
I would omit the "KB" word. It may not be clear to everyone, and also, it doesn't matter here. You might also consider a shorter title:
"Keep the Window (content) in the DOM while closed"